airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -473,18 +473,29 @@ def run_with_configuration(self, configuration):
         job = jobs.get(projectId=self.project_id, jobId=job_id).execute()
```

Comment:
Do you still need this if you are running it again at the beginning of the `try`?